Gemfile.lock

```diff
@@ -154,10 +154,12 @@ GEM
     ffi (1.15.5)
     globalid (1.1.0)
       activesupport (>= 5.0)
-    govuk-components (3.3.0)
+    govuk-components (3.1.1)
```

Comment:
This is going backwards? Is that right?

Reply:
Ooh I didn't even notice! No that's not right

Reply:
Oh no 
```
Bundler could not find compatible versions for gem "pagy":
  In Gemfile:
    pagy (~> 6.1)

    govuk-components (~> 3.3) was resolved to 3.3.0, which depends on
      pagy (~> 5.10.1)
```
Looks like we have to hold off updating pagy until govuk-components catches up?

Reply:
Aha! That is what I was worried about when pagy came in via Renovate!

Okay, shall pin Pagy to the compatible version, we can say why in the commit?